### DISCUSSION TITLE
[Question][Hw0-1] Visual Studio Code "No config found"

### DISCUSSION THREAD

**Discussion by nybunny** [prosyslab-classroom/cs348-information-security#640] [Q&A]
Name: 이나연

Why does this error pop up and how can I fix it?
![1](https://user-images.githubusercontent.com/47998214/221748767-ea148a60-a999-400b-802a-dc361fbbe853.jpg)
![2](https://user-images.githubusercontent.com/47998214/221749261-85a89ad9-1546-4497-b996-d0fd2ce6cac1.jpg)

^ These are screenshots of the files from Docker. This doesn't happen in files from Windows.

**Comment by m-spitfire** [comment]
please read the error message, and try calling `dune build` from terminal, it should solve the issue.
